src/D2L.CodeStyle.Analyzers/Async/Generator/SyntaxTransformer.cs

```diff
@@ -11,13 +11,16 @@ namespace D2L.CodeStyle.Analyzers.Async.Generator;
 /// </summary>
 public abstract class SyntaxTransformer {
 	protected readonly SemanticModel m_model;
+	protected readonly CancellationToken m_token;
+
 	private readonly ImmutableArray<Diagnostic>.Builder m_diagnostics = ImmutableArray.CreateBuilder<Diagnostic>();
 
 	/// <summary>
 	/// A name for the generator that will be shown to the user if we crash
 	/// </summary>
-	public SyntaxTransformer( SemanticModel model ) {
+	public SyntaxTransformer( SemanticModel model, CancellationToken token ) {
 		m_model = model;
+		m_token = token;

```

Comment:
Taken but not used?

Reply:
Yeah, it's used below (although to no effect currently, at least in NUnit. Maybe in VS 🤷)